src/utils/auth-util.ts

```diff
@@ -0,0 +1,24 @@
+import { AccessToken } from '../types';
+import {
+  getFromStorage,
+  removeFromStorage,
+  setToStorage,
+} from './storage-util';
+
+const ACCESS_TOKEN: string = 'access-token';
+
+export const setAccessToken = (accessToken: AccessToken): void => {
```

Comment:
Create custom react hook `useAuthContext` to be `reacty` and expose these functions

Reply:
implemented